app/views/users/show.html.erb

```diff
@@ -1,9 +1,11 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <%= invite_friend(@user) %>
   <article class="timeline">
     <h3>Recent posts:</h3>
     <ul class="posts">
       <%= render @posts %>
```

Comment:
I should only see the posts of people I'm friends with. Please add a validation for that. [Check requirement.](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friends_timeline.png)